capsules/classifier_face_emotion_openvino/meta.conf

```diff
@@ -0,0 +1,2 @@
+[about]
+api_compatibility_version = 0.2
```

Comment:
Nitpick: new line at end of file

Reply:
For reference: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file